osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -327,9 +330,9 @@ protected override void Update()
             const float velocity_adjust_cutoff = 0.98f;
             const float paused_velocity = 0.5f;
 
-            if (Beatmap.Value.Track.IsRunning)
+            if (musicController.CurrentTrack.IsRunning)
```

Comment:
.

Reply:
Not sure about this one. Shouldn't the logo be responding to the current track, not the beatmap? it's not using the beatmap elsewhere.

Reply:
This one can stay as is too.